SHFB/Source/MRefBuilder/CCI/MemoryMappedFile.cs

```diff
@@ -21,10 +22,6 @@ unsafe internal sealed class MemoryCursor
         private byte* pb;
         readonly internal int Length;
 
-        internal MemoryCursor(MemoryMappedFile/*!*/ memoryMap) : this(memoryMap.Buffer, memoryMap.Length)

```

Comment:
Probably bring this back and just skip the path that requires the P/Invoke when not on Windows